xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/ServerInfoIT.java

```diff
@@ -0,0 +1,52 @@
+package org.xrpl.xrpl4j.tests;
+
+import okhttp3.HttpUrl;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xrpl.xrpl4j.client.JsonRpcClientErrorException;
+import org.xrpl.xrpl4j.client.XrplClient;
+import org.xrpl.xrpl4j.model.client.serverinfo.ServerInfo;
+import org.xrpl.xrpl4j.model.client.serverinfo.ServerInfoType;
+
+import java.util.concurrent.TimeUnit;
+
+public class ServerInfoIT {
+
+  Logger logger = LoggerFactory.getLogger(this.getClass());
+
+  @Timeout(value = 3, unit = TimeUnit.HOURS)
+  @Test
+  @Disabled
+  public void testServerInfoAcrossAllTypes() throws JsonRpcClientErrorException, InterruptedException {
```

Comment:
Want to add a javadoc to this test just to remind us of its purpose in the future?